dali/base/dasds.cpp

```diff
@@ -8116,8 +8139,9 @@ void CCovenSDSManager::handleNodeNotify(notifications n, CServerRemoteTree &tree
     }
 }
 
-void CCovenSDSManager::handleNotify(CSubscriberContainerBase &subscriber, MemoryBuffer &notifyData)
```

Comment:
pedantic:  I could argue this should still be a reference parameter since it can never be NULL.